client/components/AdminPanel.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+
+import UserTile from './UserTile';
+import UserModal from './UserModal/UserModal';
+import Pagination from './Pagination/Pagination';
+import { fetchUsers, adminCreateUser, updateUser, deleteUser } from '../actions/actions';
+
```

Comment:
This is the `<AdminPanel />`, definitely the beefiest component in the app. It renders the users from the redux state, allows the admin to create new users, and edit and delete current ones. It also renders the Pagination bar and allows the admin to filter the results